Codeinwp/neve
# fix(customizer): append plan to woocomerce up sell labels

### Summary


The Customizer upsell for the WooCommerce booster now includes the plan names to eliminate the confusion for users who purchased personal plan and expected all the features.

### Will affect visual aspect of the product

YES

### Screenshots 

![image](https://github.com/Codeinwp/neve/assets/17597852/02653923-bda7-46d4-b181-00c3a9643cc8)


### Test instructions


- Install WooCommerce
- Using Neve Free, in Customizer on the WooCommerce tab, check if the upsell labels of the Neve PRO plugin include the plan names

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/neve-pro-addon/issues/2732
